modules/scripts/timeline.py

```diff
@@ -0,0 +1,50 @@
+from docutils import nodes
+
+from docutils.parsers import rst
+
+class timeline(nodes.General, nodes.Element):
```

Comment:
The documentation says to have blank class to inherit from the standard docutils. That's why i used http://www.sphinx-doc.org/en/stable/extdev/tutorial.html